code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -527,9 +527,11 @@
 		"r_foot" = list("path" = /obj/item/organ/external/foot/right/ipc)
 		)
 
-	abilities = list(/mob/living/carbon/human/proc/change_monitor,)
```

Comment:
I assume you are getting `src = client` from a runtime error
but runtime errors can fuck up their reported src/usr/loc